lib/ical/types.js

```diff
@@ -21,6 +22,21 @@ import Time from "./time";
  * @memberof ICAL.Time
  */
 
+/**
+ * @typedef {Object} timeInit         Time initialization
+ * @property {Number=} year           The year for this date
+ * @property {Number=} month          The month for this date
+ * @property {Number=} day            The day for this date
+ * @property {Number=} hour           The hour for this date
+ * @property {Number=} minute         The minute for this date
+ * @property {Number=} second         The second for this date
+ * @property {Boolean=} isDate        If true, the instance represents a date
+ *                                    (as opposed to a date-time)
+ * @property {String=} timezone       Timezone id if zone is unknown [internal]
+ * @property {Timezone=} zone         Resolved timezone object [internal]
```

Comment:
I added those because they are (only) used in `Time.fromData()`. They weren't documented before though.

This typedef (`timeInit`) is not exported anyway so I think the internal members can be kept at least for internal documentation purposes.